src/include/sof/platform.h

```diff
@@ -54,6 +54,14 @@ int platform_context_save(struct sof *sof);
 
 /** @}*/
 
+#ifdef CONFIG_IMX8ULP
+#define convert_host_to_local(addr) (SDRAM0_BASE + ((addr) - MEM_RESERVED))
+#define convert_local_to_host(addr) (MEM_RESERVED + ((addr) - SDRAM0_BASE))
+#else
+#define convert_host_to_local(addr) (addr)
+#define convert_local_to_host(addr) (addr)
+#endif
+
```

Comment:
This is good for now, but we should consider a better way for future changes. For the moment I'll approve as-is.